web/src/beta/lib/core/mantle/types/index.ts

```diff
@@ -26,12 +26,14 @@ export type LayerSimple = {
   properties?: any;
   defines?: Record<string, string>;
   events?: Events;
+  hidden?: boolean;
 } & Partial<LayerAppearanceTypes> &
   LayerCommon;
 
 export type LayerGroup = {
   type: "group";
   children: Layer[];
+  hidden?: boolean;
```

Comment:
Also by the way, I think we can use `visible` instead of `hidden`.
https://github.com/reearth/reearth/blob/787094e057884b96eee86982981ef9f909f17e01/web/src/beta/lib/core/mantle/types/index.ts#L43